.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+    "endOfLine": "crlf",
```

Comment:
Why is the EOL set to CRLF? Also why is tabWidth set to 4? Any specific reasons for doing so? (btw first time using this review feature of Github so bear with me please)

Reply:
- CRLF is just the carriage return character.
- I just copied my standard code prettier config, its been changed to 2. 
Btw you need the prettier extension installed in VScode (also make sure to have the format on save option installed).